resilience4j-circuitbreaker/src/main/java/io/github/resilience4j/circuitbreaker/CircuitBreakerConfig.java

```diff
@@ -100,8 +104,12 @@ public int getSlidingWindowSize() {
         return slidingWindowSize;
     }
 
-    public Predicate<Throwable> getRecordFailurePredicate() {
```

Comment:
May we can add deprecated instead and let it return recordExceptionPredicate as well to support backward compatible code with cb config object  ?

Reply:
Actually it's dangerous, because the implementation has also changed. I think it should be even a non public method or?

Reply:
It's a non backward compatible change.